changelog_unreleased/css/pr-8535.md

```diff
@@ -0,0 +1,20 @@
+#### Incorrect whitespace added after escaped colon in css grid line name ([#8535](https://github.com/prettier/prettier/pull/8535) by [@boyenn](https://github.com/boyenn))
+
+<!-- prettier-ignore -->
+```css
+/* Input */
+.grid {
+  grid-template-rows:
+    [row-1-00\:00] auto;
+}
+
+/* Prettier stable */
+.grid {
+  grid-template-rows: [row-1-00\: 00] auto;
```

Comment:
Wasn't sure if I should remove the extra `\` generated in the tests. Took an educated guess and I did, but let me know if that's not right.